### DISCUSSION TITLE
Add option to filter custom generated documentation in the built-in documentation

### DISCUSSION THREAD

**Discussion by azur-wolve** [godotengine/godot-proposals#8569] [Editor]
**[Edited: see → https://github.com/godotengine/godot-proposals/issues/4285]**

In Search Help (`F1`), have a "custom only" checkbox to only show custom documentation.
![image](https://github.com/godotengine/godot-proposals/assets/88264884/97db0ad7-cd05-40ac-9c33-54ebd35e74ee)

Useful for large projects and teams.

**Comment by dalexeev** [comment]
* See #4285.
